### ISSUE TITLE
Output of check_smtp is not consistent

### ISSUE THREAD

**Issue by riskersen** [monitoring-plugins/monitoring-plugins#1380]
Some outputs in check_smtp are like "STATUS - Output" and some like "Output"

Despite the fact that its redundant to write WARNING, CRITICAL, .. in front of the output, it should be consistent.

**Comment by riskersen** [comment]
In IRC channel we discussed if we drop "SERVICE STATUS - " (e.g. CRITICAL - , WARNING -) in output because it should be printed by GUIs.

What to do? I would fix this output but we must decide whats the future way.

**Comment by weiss** [comment]
> What to do? I would fix this output but we must decide whats the future way.

If you'd like to fix this right now, I'd let `check_smtp` spit out `SERVICE STATUS - [...]` consistently, as that's the format currently emitted by most other plugins.  We can discuss changing the output format (or making it configurable) for all plugins at a later point in time.

**Comment by sni** [comment]
I definitly like the SERVICE STATUS - OUTPUT|PERF format most. From time to time i have to run
the plugins manually from the command line and it definitly helps. Also, like Holger already said, this
is the most common format.
When it comes to GUIs, i could imagine GUIs removing that state if they choose a color instead, but
something the GUI should do, not the plugins.

**Comment by weiss** [comment]
> I definitly like the SERVICE STATUS - OUTPUT|PERF format most.

It seems I'm the only one who [doesn't like it][1] :-)  I should just [make this configurable][2] with a command-line flag one day, without changing the current format by default.

> From time to time i have to run the plugins manually from the command line and it definitly helps.

I see your point for the `STATUS` part, but the `SERVICE` part helps as well?

[1]: https://www.monitoring-plugins.org/archive/devel/2007-July/006121.html
[2]: https://github.com/monitoring-plugins/monitoring-plugins/commit/006597a12d28d9cd752b3eeb56dff3bc97049f4c

**Comment by riskersen** [comment]
Do you mean for example "check_smtp WARNING - Foobar|asdf=1" ?

**Comment by weiss** [comment]
> Do you mean for example "check_smtp WARNING - Foobar|asdf=1" ?

`SMTP WARNING - Foobar|asdf=1`

**Comment by riskersen** [comment]
Okay.. i agree with you, that is too much output. 
Imho is STATUS - Output|perf the best option.

**Comment by weiss** [comment]
> Imho is STATUS - Output|perf the best option.

Yes, personally I'd let plugins use that format on the command line and I'd omit the `STATUS` when called from Icinga/whatever.

An option that makes this configurable should make everybody happy (I hope).

**Comment by RincewindsHat** [comment]
Release 3.0.0 changed the output significantly. This issue is either outdated or fixed now :-)
